src/routes/home/components/__tests__/DisciplineCard.spec.tsx

```diff
@@ -20,7 +21,9 @@ describe('DisciplineCard', () => {
 
   const renderDisciplineCard = (): RenderAPI =>
     render(
-      <DisciplineCard disciplineId={mockDisciplines()[0].id} navigateToNextExercise={navigate} onPress={onPress} />
+      <NavigationContainer>
```

Comment:
Why is this needed?

Reply:
DisciplineCard uses useNavigation hook, which will throw an error if no NavigaitonContainer is provided.